schemas/search-criteria.json

```diff
@@ -0,0 +1,27 @@
+{
+    "type": "array",
+    "decription": "Array of objects, each of which represents a search to be performed by the search layer.",
+    "items": {
+        "type": "object",
+        "properties": {
+            "field": {
+                "type": "string",
+                "description": "Field to search."
+            },
+            "value": {
+                "type": "string",
+                "description": "Value to search."
+            },
+            "search_type": {
+                "$ref": "pyshelf.search.type.Type",
```

Comment:
Yeah, I suppose I would like them to be valid jsonschema even if we are not validating against them